crates/core/Cargo.toml

```diff
@@ -11,11 +11,12 @@ categories = ["network-programming"]
 edition = "2021"
 
 [dependencies]
+kitsune2_api = { workspace = true }
```

Comment:
These are easier to find if they're kept in a separate block. We've done that in most places I think